$OpenBSD$

clang does not like container_of(), upstream appears to know this.

Index: mk/build.mk
--- mk/build.mk.orig
+++ mk/build.mk
@@ -53,7 +53,7 @@ endif
 # Clang does not like container_of()
 ifneq "$(CC)" "clang"
 ifneq "$(uname_S)" "Darwin"
-  WARNINGS += -Wcast-align
+  WARNINGS += -Wno-cast-align
 endif
 endif
 
